src/components/organisms/UiMenu/_internal/UiMenuItem.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <UiListItem
+    class="ui-menu-item"
+    v-bind="listItemAttrs"
+  >
+    <UiButton
+      :class="['ui-button--outlined', 'ui-menu-item__button']"
+      v-bind="$attrs"
+    >
+      <!-- @slot Use this slot to replace label template -->
+
+      <slot
+        name="label"
+        v-bind="labelClass"
+      >
+        <UiText
+          :class="['ui-menu-item__label', labelClass]"
+        >
+          <!-- @slot Use this slot to place label content inside menu-item -->
+          <slot />
+        </UiText>
+      </slot>
+      <!-- @slot Use this slot to replace suffix template -->
+      <slot
+        v-if="hasSuffix"
```

Comment:
```suggestion
```
R: Usually we shouldn't use `v-if` on slots. User should always put their own template to slot and include their own functionality.

```
<slot ...>
  <template v-if="true">
 </template>
</slot>
```